test/python/test_bridge_core.py

```
@@ -50,6 +57,268 @@ def test_plain_values_do_not_import_scientific_codecs() -> None:
+        allow_nan=False,
+    )
+
+    assert nested_json == f'{{"outer": {{"values": {root_json}}}}}'
+
+
+def test_model_dump_information_error_preserves_origin_behavior() -> None:
+    class BrokenModel:
+        def model_dump(self, **_kwargs: object) -> object:
+            raise ValueError('invalid information')
+
+    with pytest.raises(RuntimeError, match='^model_dump failed: invalid information$'):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)